src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -1,47 +1,156 @@
 package core.basesyntax;
 
 import java.util.List;
+import java.util.Objects;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
     @Override
```

Comment:
Private methods and classes should be after public ones in your class. Consider moving the `Node` class definition to the end of the `MyLinkedList` class.